experimental/devices/as7262/as7262.go

```diff
@@ -284,13 +298,9 @@ func (d *Dev) Gain(gain Gain) error {
 	d.mu.Lock()
 	defer d.mu.Unlock()
 
-	done := make(chan struct{}, 1)
-	d.canceledMu.Lock()
-	d.done = done
-	d.canceled = false
-	d.canceledMu.Unlock()
-
-	if err := d.writeVirtualRegister(controlReg, uint8(gain)); err != nil {
+	ctx, cancel := context.WithTimeout(context.Background(), sensorTimeout)
```

Comment:
In this case, I'd say to just use context.Background(), because you don't intent to cancel the context while the function is running.

Reply:
Very true, fixed.